libultraship/libultraship/Window.cpp

```diff
@@ -40,7 +44,7 @@ extern "C" {
         }
 
         // TODO: This for loop is debug. Burn it with fire.
-        for (size_t i = 0; i < SDL_NumJoysticks(); i++) {
+        for (int i = 0; i < SDL_NumJoysticks(); i++) {
```

Comment:
Everywhere else you replaced `int` with `size_t`, except here. Why?

Reply:
This is because SDL chose to use signed types for counts for some reason.